infra/base-images/base-runner/test_all

```diff
@@ -56,7 +56,7 @@ for FUZZER_BINARY in $(find $OUT/ -maxdepth 1 -executable -type f); do
   LOG_PATH_FOR_BROKEN_TARGET="${BROKEN_TARGETS_DIR}/${FUZZER}"
 
   # Launch bad build check process in the background.
-  bad_build_check $FUZZER_BINARY > $LOG_PATH_FOR_BROKEN_TARGET &
+  bad_build_check $FUZZER_BINARY &> $LOG_PATH_FOR_BROKEN_TARGET &
```

Comment:
The purpose of this change was to fix the issue where bad_build_check would error out but test_all wouldn't realize.